src/lib.rs

```diff
@@ -1,5 +1,6 @@
 #![deny(missing_docs)]
 #![allow(stable_features)]
+#![cfg_attr(docsrs, feature(doc_auto_cfg))]
```

Comment:
Hah, I didn't even know about `doc_auto_cfg`, I was planning on doing this the Very Hard Way 😅